webmacs/minibuffer/__init__.py

```diff
@@ -335,3 +351,13 @@ def close_prompt(self):
 
     def _prompt_closed(self):
         self._prompt = None
+
+    @classmethod
+    def update_rlabel(cls):
+        from .. import windows, BUFFERS
+        for window in windows():
+            window.minibuffer().rlabel.setText(
+                variables.get_variable("minibuffer-right-label").value % dict(
```

Comment:
MINIBUFFER_RIGHTLABEL.value would be a bit more effective and will have the same effect.